src/components/Table/TableRow.tsx

```diff
@@ -26,7 +25,10 @@ export type TableSortFunction<T> = (a: T, b: T) => number;
 export interface TableColumn<T> {
 	cellAttributes?:
 		| React.AnchorHTMLAttributes<HTMLAnchorElement>
-		| ((value: any, row: T) => React.AnchorHTMLAttributes<HTMLAnchorElement>);
+		| ((
+				row: T,
+				value: T[keyof T],
```

Comment:
With the switch away from `get` typescript was able to point out that these typings were wrong, so I've correctly typed them and switched the names around to represent what they actually are. So this doesn't change the existing runtime behavior and instead just fixes the typings